resources/intent/fokoos_base/abs/fokoos_abs_nozzle_0.40_layer_0.10_visual.inst.cfg

```diff
@@ -0,0 +1,54 @@
+[general]
+version = 4
+name = Visual
+definition = fokoos_base
+
+[metadata]
+setting_version = 20
+type = intent
+quality_type = fokoos_layer_010
+intent_category = FKS Fast
```

Comment:
I'd strongly suggest using "quick" as a type here, as that means it will also be translated and have correct icons in the interface.

Reply:
The suggest is OK.